.travis.yml

```diff
@@ -8,10 +8,10 @@ os:
 cache:
```

Comment:
Can we do just Node 8 and 10?  6 seems to have issues on Travis' linux environment.

Reply:
I think 8 and 10 would be OK, since that is what is recommended: https://nodejs.org/en/